org.upbge.UPBGE.yml

```
@@ -0,0 +1,79 @@
+id: org.upbge.UPBGE
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: blender
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--filesystem=host"
+  - "--filesystem=/run/spnav.sock:ro"
+  - "--env=SPNAV_SOCKET=/run/spnav.sock"
+  - "--env=TMP_DIR=/tmp"
+  - "--env=TMP=/tmp"
+add-extensions:
+  org.upbge.UPBGE.Codecs:
+    directory: lib/codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: "."
+cleanup-commands:
+  - mkdir -p /app/lib/ffmpeg
+modules:
+  - shared-modules/libdecor/libdecor-0.1.1.json
+  - name: x264
+    config-opts:
+      - "--prefix=/app/lib/codecs"
+      - "--enable-lto"
+      - "--enable-pic"
+      - "--enable-shared"
+      - "--disable-cli"
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: baee400fa9ced6f5481a728138fed6e867b0ff7f
+    cleanup:
+      - "/lib/codecs/include"
+      - "/lib/codecs/lib/pkgconfig"
+  - name: upbge
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 blender.sh /app/bin/blender
+      - mv blender /app/blender
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 /app/blender/upbge.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 /app/blender/upbge-symbolic.svg /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg
+      - cd /app/blender/*.*/datafiles && mv locale /app/share/locale && ln -sf /app/share/locale locale
+      - install -Dm644 $FLATPAK_ID.appdata.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: archive
+        url: https://github.com/UPBGE/upbge/releases/download/v0.36.1/upbge-0.36.1-linux-x86_64.tar.xz
+        sha256: 0d53fdace0f9755c3475e5cb98dfee3e63f9968aa2b7fe516b0a8c4b1326ca4f
+        strip-components: 0
+      - type: shell
+        commands:
+          - mv upbge*linux* blender
+      - type: script
+        commands:
+          - cd /app/blender
+          - exec ./blender "$@"
+        dest-filename: blender.sh
+      - type: file
+        url: https://raw.githubusercontent.com/UPBGE/upbge/v0.36.1/release/freedesktop/org.upbgeteam.UPBGE.appdata.xml
+        sha256: ce481e326de27b9f73350f1c48ea6854abe8c7c44b211f340918c6d211f85b01
+      - type: file
+        path: org.upbge.UPBGE.desktop
+        sha256: 3296656960080be4eac9458c908dc61d1250f4b5634c2922cb4a7da70e6fef62
+      - type: file
+        path: org.upbge.UPBGE.appdata.xml
+        sha256: 344efe06b4d7e285d0361b4c279eb53ab1d85e6c0a1b8b157fab91a5c6d4864f
```

```suggestion
      - type: file
        path: org.upbge.UPBGE.appdata.xml
```

local file don't need checksums.